mne/coreg.py

```diff
@@ -1963,3 +1968,67 @@ def reset(self):
         self._extra_points_filter = None
         self._update_nearest_calc()
         return self
+
+    def _get_fiducials_distance(self):
+        transformed_mri_lpa = apply_trans(self._mri_trans, self._lpa)
+        transformed_hsp_lpa = apply_trans(
+            self._head_mri_t, self._dig_dict['lpa'])
+        lpa_distance = np.linalg.norm(
+            np.ravel(transformed_mri_lpa - transformed_hsp_lpa))
+
+        transformed_mri_nasion = apply_trans(self._mri_trans, self._nasion)
+        transformed_hsp_nasion = apply_trans(
+            self._head_mri_t, self._dig_dict['nasion'])
+        nasion_distance = np.linalg.norm(
+            np.ravel(transformed_mri_nasion - transformed_hsp_nasion))
+
+        transformed_mri_rpa = apply_trans(self._mri_trans, self._rpa)
+        transformed_hsp_rpa = apply_trans(
+            self._head_mri_t, self._dig_dict['rpa'])
+        rpa_distance = np.linalg.norm(
+            np.ravel(transformed_mri_rpa - transformed_hsp_rpa))
```

Comment:
can you use a for loop to avoid the big copy/paste? maybe rely on an  internal tiny function?

Reply:
I'll refactor it